site/src/pages/TaskPage/TaskStatusLink.tsx

```diff
@@ -30,15 +30,21 @@ export const TaskStatusLink: FC<TaskStatusLinkProps> = ({ uri }) => {
 					switch (type) {
 						case "pull":
 							icon = <GitPullRequestArrowIcon />;
-							label = number
-								? `${org}/${repo}#${number}`
-								: `${org}/${repo} pull request`;
+							label =
+								number === "new"
```

Comment:
Drive-by fix: Handle new. I personally ran into this where open a new PR was called `coder/coder#new`, very confusing.